docs/building_the_project_with_dpcpp.rst

```diff
@@ -225,6 +225,21 @@ A few often-used architectures are listed below:
 For a host with ROCm installed, the device architecture can be retrieved via the
 ``rocminfo`` tool. The architecture will be displayed in the ``Name:`` row.
 
+.. _build_for_other_SYCL_devices:
+
+Building for other SYCL devices
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SYCL enables portable heterogeneous computing on a wide range of accelerators.
+Consequently, it is possible to use oneMKL Interfaces with accelerators not
+anticipated by the oneMKL Interfaces team.
+
+For generic SYCL devices, only portBLAS and portFFT backend are enabled. The user must
+set the appropriate ``-fsycl-targets`` for their device, and also any
+other option required for performance. See
+`Building for portBLAS`_ and `Building for portFFT`_. Extensive testing is strongly advised for these
+unsupported configurations.
```

Comment:
Can you reformat the paragraph to break the lines at 80 columns? The rewrap extension is great for that if you use VS Code.